bindings/bindings-linux-aarch64-R4.4-devel.rs

```diff
@@ -639,7 +639,6 @@ pub const Rboolean_FALSE: Rboolean = 0;
 #[doc = ", MAYBE"]
 pub const Rboolean_TRUE: Rboolean = 1;
 pub type Rboolean = ::std::os::raw::c_uint;
-#[doc = "This definition uses an anonymous structure, which is defined in C11 (but\nnot C99).  It is, however, supported at least by GCC, clang and icc.  The\nprivate_data_c member should never be used in code, but tells the compiler\nabout type punning when accessing the .r and .i elements, so is safer to use\nwhen interfacing with Fortran COMPLEX*16 or directly C99 _Complex double\n(PR#18430).\n\nThis form of static initialization works with both definitions:\nRcomplex z = { .r = 1, .i = 2 };\n\nAnonymous structures and C99 _Complex were not incorporated into C++\nstandard.  While they are usually supported as compiler extensions, warnings\nare typically issued (-pedantic) by a C++ compiler."]
```

Comment:
Why is this now gone from some of the bindings? Is it because the version has changed?

Reply:
For reference:
https://github.com/r-devel/r-svn/blob/master/src/include/R_ext/Complex.h

See the history. 

This is unfortunately not an exact science, and these comments are not put there to be found my automatic tools like `bindgen`, so if they disappear, I cannot but be a little 🤷‍♂️ about it.